dll/shellext/acppage/pot/cs-CZ.po

```diff
@@ -0,0 +1,108 @@
+#. extracted from lang/en-US.rc, lang/cs-CZ.rc
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"PO-Revision-Date: 2010-05-25\n"
+"Last-Translator: Radek Liska aka Black_Fox\n"
+"Language-Team: radekliska at gmail dot com\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Accelerator-Marker: &\n"
+"X-Generator: Translate Toolkit 2.2.4\n"
+"X-Merge-On: location\n"
+
+#: DIALOGEX.IDD_ACPPAGESHEET.CAPTION
+msgctxt "DIALOGEX.IDD_ACPPAGESHEET.CAPTION"
+msgid "Compatibility"
+msgstr "Kompatibilita"
+
+#: DIALOGEX.IDD_ACPPAGESHEET.LTEXT.IDC_EXPLANATION
+msgid ""
+"If you have problems running this program in ReactOS, select an operating "
+"system ReactOS should report to the application.\n"
+"Please note that this is a work in progress."
+msgstr "Pokud jsou problémy se spouštěním tohoto programu v systému ReactOS, zvolte operační systém, který ReactOS tomuto programu sdělí.\nPlease note that this is a work in progress."
+
+#: DIALOGEX.IDD_ACPPAGESHEET.GROUPBOX.IDC_COMPATGROUP
+msgid "Compatibility mode"
+msgstr "Režim kompatibility"
+
+#: DIALOGEX.IDD_ACPPAGESHEET.CHECKBOX.IDC_CHKRUNCOMPATIBILITY
+msgid "Run this program in compatibility mode for:"
+msgstr "Spustit tento program v režimu kompatibility pro:"
+
+#: DIALOGEX.IDD_ACPPAGESHEET.GROUPBOX.Display_settings
+msgid "Display settings"
+msgstr "Nastavení zobrazení"
+
+#: DIALOGEX.IDD_ACPPAGESHEET.CHECKBOX.IDC_CHKRUNIN256COLORS
+msgid "Run in 256 colors"
+msgstr "Spustit v 256 barvách"
+
+#: DIALOGEX.IDD_ACPPAGESHEET.CHECKBOX.IDC_CHKRUNIN640480RES
+msgid "Run in 640x480 screenresolution"
```

Comment:
Hmm. So if you fixed the English version to say "screen resolution" here, you'd still need to update all the other po's?

Reply:
No, because the messageID is changed, all translations will no longer match.
After editing the 'master' (or template, or pot) file, you'd typically run a tool like msgmerge  or poedit's `update from template`